### PR TITLE
 [MAINT]- Docs theme improvement

### PR DESCRIPTION
**Pull request by trallard**
There are some accessibility fails in the current docs - as well as general bad practices with how URLs are displayed. 
This PR aims to make the theme more cohesive and better aligned with WCAG accessibility standards.

- :wheelchair: Enhance accessibility
- :lipstick: Update overall docs style

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
...
+## Installing `node`
+
+- Checking if node is already present in the system (should print the version number), from the command line:
+
+  ```bash
+   node -v
+  ```
+
+- To install node visit [nodejs.org](https://nodejs.org/en/download/)
+
```
Comment (kcpevey) [inline docs/README.md:14]: node is required for yarn? The install is all yarn based.

**Review by kcpevey — COMMENTED** [review]
We had moved away from having an `src` directory and just had a `css` directory instead. What's the idea behind going back to `src` to just have the css in there?

File: docs/README.md (hunk 1 above)
Comment (trallard) [reply docs/README.md:14]: Yarn is a package manager for node so you definitely need to install node and yarn first before installing all the dependencies

**Comment by trallard** [comment]
I cannot remember when / why we made that change. 
But in general having one single (really long) CSS is not best practice. 
I am adhering to a more "react-like" structure here where the components and general theming are distributed across scss files.
Also I am favouring using scss directly over CSS as the former is way more flexible and readable than the latter

File: docs/README.md (hunk 1 above)
Comment (trallard) [reply docs/README.md:14]: Also ignore this readme - you made changes recently so I did not update stuff here 👍🏽

**Comment by kcpevey** [comment]
I have no argument against breaking up the long css file (I think it was auto-generated from the docusaurus template). I also have no argument on `scss` vs `css` - I didn't even know `scss` was a thing... 

My comment was only directed at the folder in which they were kept.

**Comment by trallard** [comment]
The current folder structure better aligns with js/React conventions

**Comment by trallard** [comment]
SO managed to fix the deploy - 😶‍🌫️  forgot yarn complains with license files sometimes

**Pull request merged**